cs/random.go

```diff
@@ -15,3 +19,48 @@ type rng interface {
 	// swap swaps the elements with indexes i and j.
 	Shuffle(n int, swap func(i, j int))
 }
+
+var modifiedNormalDistribution [126]float64
+var initialized = false
+
+func normalDistribution(μ float64, σ float64, x float64) float64 {
+	var eNum = -math.Pow(x-μ, 2)
+	var eDen = 2 * math.Pow(σ, 2)
+
+	return math.Exp(eNum/eDen) / math.Sqrt(2*math.Pi*math.Pow(σ, 2))
+}
+
+func initRandom() {
```

Comment:
is this something that only happens once a program or is it unique per game? It looks once a program...

If so, golang has an init function you can put in a package that runs once during startup or load (not sure exactly when).

https://go.dev/doc/effective_go#init